### ISSUE TITLE
geolocator_windows missing Nuget.exe

### ISSUE THREAD

**Issue by barbalex** [Baseflow/flutter-geolocator#963]
## 🐛 Bug Report
I am using geolocator_windows. I tried to install it as mentioned [here](https://github.com/Baseflow/flutter-geolocator/issues/840#issuecomment-1020495588): simply added the package in pubspec.yaml.

But when running on windows this errors:
```
Launching lib\main.dart on Windows in debug mode...
lib\main.dart:1
Nuget.exe not found, trying to download or use cached version.
CMake Error at flutter/ephemeral/.plugin_symlinks/geolocator_windows/windows/CMakeLists.txt:33 (message):
  Failed to install nuget package Microsoft.Windows.CppWinRT.2.0.210806.1


Exception: Unable to generate build files
Exited (sigterm)
```

I tried to install nuget but as it seems to be only an exe it is not too clear how this should happen.

**Comment by mvanbeusekom** [comment]
Hi @barbalex, 

Thank you for submitting this issue, we will have a look at how we can improve this installation process. 

As a workaround you could download the `nuget.exe` binary (which you already did. Save the `nuget.exe` binary somewhere to disk (personally I keep it in a folder `C:\Tools\NuGet\`) and add this folder to your `PATH` environment variable. When `NuGet.exe` is available on your path you should be able to build.

**Comment by barbalex** [comment]
Hm. I think I found the problem. Though knowing nothing about visual studio I may be wrong.

After removing geolocator_windows I did not manage to get the project starting again. Got many errors, a lot of them mentioning visual studio 2019.

So I opened visual studio and let it run a bunch of updates. Then I restarted and found out that it needed a license and that it had run out. A few clicks later the license seemed to have been renewed (why this dance about a license that seems to cost nothing anyway, microsoft?).

A windows restart and `flutter clean` followed by `flutter pub get` later everything works now.

So it seems this was just collateral damage caused by windows build tools. Feel free to close the issue.
